app/views/activities/show.html.erb

```diff
@@ -2,37 +2,57 @@
-  <%= image_tag(@activity.image_url, class:"w-100 activity-image") %>
-  <div class="container mb-5">
-    <div class="mt-2">
-      <span class="badge rounded-pill bg-info text-dark"><%= @activity.age_group %></span>
-      <span class="badge rounded-pill bg-info text-dark"><%= @activity.category.name.titleize %></span>
-    </div>
-    <div class="mt-2 d-flex justify-content-between activity-title">
-      <span><%= @activity.title %></span>
-      <div class="d-flex">
-        <span><%= button_to "", "#", class:"border-0 fa fa-share-alt bg-transparent" %></span>
-
-        <%= render "wishlists/wishlist_button", activity: @activity, wishlist: @wishlist %>
-        <%= render "heart_button", activity: @activity, heart: @heart %>
+<div class="container">
+  <div class="row">
+    <%= image_tag(@activity.image_url, class:"w-100 activity-image g-0") %>
+    <div class="col">
+      <div class="page-container">
+        <span class="badge rounded-pill bg-info text-dark"><%= @activity.age_group %></span>
+        <span class="badge rounded-pill bg-info text-dark"><%= @activity.category.name.titleize %></span>
+        <div class="mt-4 d-flex justify-content-between activity-title">
+          <h2><%= @activity.title %></h2>
+          <div class="d-flex">
+            <%= render "share_button" %>
+            <%= render "wishlists/wishlist_button", activity: @activity, wishlist: @wishlist %>
+            <%= render "heart_button", activity: @activity, heart: @heart %>
+          </div>
+        </div>
+        <div class="mt-3">
+          <% if @activity.average_rating.to_f > 0 %>
+            <em>Average rating: <%= @activity.average_rating.ceil(1) %> / 5</em>
+          <% else %>
+            <em>Average rating: Nil</em>
+          <% end %>
+        </div>
+        <div class="mt-3">
+          <h4>Description:</h4>
+          <%= @activity.description %>
+        </div>
+        <div class="mt-3">
+          <h4 class=>Reviews:</h4>
+            <% if @activity.reviews.count > 0 %>
+              <% @activity.reviews.first(3).each do |review| %>
+                <p><strong><%= review.user.username.capitalize %></strong>: <%= review.content %> (Rating: <%= review.rating %>)</p>
+                <% if review.user_id == current_user.id %>
+                  <%= link_to "Edit", edit_activity_review_path(@activity, review.id)  %>
+                  <%= link_to "Delete", review_path(review), data: { turbo_method: :delete, turbo_confirm: 'Delete review?' } %>
+                  <hr>
+                <% end %>
+              <% end %>
+              <div class="mt-3 text-center">
+                <% if @activity.reviews.count > 3 %>
+                  <%= link_to "See all reviews >", activity_reviews_path(@activity) %>
+                <% end %>
+              </div>
+            <% else %>
+              <p>No reviews yet.</p>
+            <% end %>
+          <div class="mt-5">
+            <h4>Share your thoughts with others!</h4>
```

Comment:
if the user has written a review, will this section be shown?

Reply:
think you helped with the conditional, thanks!